src/components/BotProfileImage.tsx

```diff
@@ -4,9 +4,35 @@ import { useTheme } from 'styled-components';
 import { useChatContext } from './chat/context/ChatProvider';
 import { getColorBasedOnSaturation } from '../colors';
 import { useConstantState } from '../context/ConstantContext';
+import { themedColors } from '../foundation/colors/css';
 import BotProfileIcon from '../icons/bot-profile-image-small.svg';
 
-const Container = styled.span<{ backgroundColor: string; size: number }>`
+function isMaybeFavicon(url: string) {
+  if (url.length < 4) return false;
+  const fileName = url.substring(url.lastIndexOf('/') + 1);
+  return /fav_|favicon|\.ico/.test(fileName);
```

Comment:
fav_|favicon these two are ok? file name could be anything

Reply:
We retrieve information from the customer's site on the Sendbird server.
Therefore, we agreed with the server to add the `fav_` prefix when saving the favicon.